src/ExpressionLanguage/ExpressionFunction/GraphQL/Relay/FromGlobalID.php

```diff
@@ -5,19 +5,19 @@
 namespace Overblog\GraphQLBundle\ExpressionLanguage\ExpressionFunction\GraphQL\Relay;
 
 use Overblog\GraphQLBundle\ExpressionLanguage\ExpressionFunction;
+use Overblog\GraphQLBundle\Relay\Node\GlobalId;
```

Comment:
Scrutinizer: This use statement conflicts with another class in this namespace, ``Overblog\GraphQLBundle\ExpressionLanguage\ExpressionFunction\GraphQL\Relay\GlobalId``. Consider defining an alias.

Reply:
This code must be old, because I already have this line using an alias:
```
use Overblog\GraphQLBundle\Relay\Node\GlobalId as GlobalIdHelper;
```

Strange.

Reply:
I don't see alias on [your branch](https://github.com/murtukov/GraphQLBundle/blob/feature/validation_v.0.13/src/ExpressionLanguage/ExpressionFunction/GraphQL/Relay/FromGlobalID.php#L8), maybe you loose the change on a rebase?